apple/RNCWebView.m

```diff
@@ -701,6 +699,12 @@ - (void)scrollViewWillBeginDragging:(UIScrollView *)scrollView
 }
 #endif // !TARGET_OS_OSX
 
+- (void)setUserAgent:(NSString*)userAgent
```

Comment:
I wonder why it was only done at view creation time before. I can't find any docs saying it's not valid to update at any point. 💯